### ISSUE TITLE
Designer overwrite when there are multiple opens (auto save behavior)

### ISSUE THREAD

**Issue by brianehlert** [octoblu/issues#19]
Here is the scenario:  You have a shared account you are using for a demo.  
Joe opens the flow on his machine to perform a number of tests.
Alice opens the flow on her desk to make some new edits.
Alice completes a number of edits, starts and stops the flow repeatedly, and tests here changes and all is good.  She closes her browser and leaves for the day.
(During all of this Joe never noticed that anything was modified, his view never changed)
Joe continues to work after Alice leaves and happens to stop and start the flow a few more times.  And he heads home.
Later Alice remembers a misspelling that she wanted to fix in the modifications that she just made.
She opens the flow in the designer, and .....  none of her edits exist .... (panic) ....  The flow is in the exact state it was before she made any edits.  In fact, this is the same state the Joe had last seen it in.

Did Joe's state auto save over Alice's edits?  Me is guessing so.
Could this be made better by a save button that triggers a reload?  (who knows).
All we know is that Alice has Joe tied in the corner and she rebooted his workstation to prevent him from opening the flow while she painstakingly adds all of her edits back in.

**Comment by royvandewater** [comment]
Right now, we only support one user logged in to an account at a time, and only one instance of a specific flow open at a time. We want to move to more of a google docs experience, where you can see each others changes live, however, it's still quite a ways out.

**Comment by brianehlert** [comment]
I take it there is no short term workaround then beyond coordination or
Tying up your teammates.

Thanks for using this as a placeholder for the feature.

**Comment by royvandewater** [comment]
Correct. The easiest way to prevent stepping on each others toes would probably be to break each flow up into smaller discrete pieces and use inter-flow communication to compose a bigger flow out of the pieces. This should also improve flow reusability.
